packages/blocks/src/block-manager/index.tsx

```diff
@@ -1,23 +1,35 @@
-import {
-	type BlockData,
-	type BlockTreeNode,
-	type BlockDefinitions,
-} from '@snapwp/core';
 import defaultBlockDefinitions from '@/blocks';
 import flatListToHierarchical from '@/utils/flat-list-to-hierarchical';
 
+export interface BlockData {
+	type: keyof typeof defaultBlockDefinitions;
+	cssClassNames?: Array< string | null > | null;
+	clientId?: string | null;
+	parentClientId?: string | null;
+	renderedHtml?: string | null;
+	attributes?: Record< any, any >;
```

Comment:
If the intent here is for attributes to get overloaded by the `{BlockComponent}` we should `unknown`, not any (and we know the key is always as string)